docs/language.rst

```diff
@@ -1757,6 +1757,60 @@ The syntax for calling external call is the same as the external call, except fo
 that it must be done on a contract type variable. Any error in an external call can
 be handled with :ref:`try-catch`.
 
+Internal calls and externals calls
+___________________________________
+
+An internal function call is one that is executed by the current contract. This
+is much more efficient than an external call. An external call requires the
+address of the contract to call, the arguments must be *abi encoded* (also known
+as serialization), then the runtime must set up the VM for the called contract
+(the callee). It must decode the arguments, and then encode return values. Lastly
+the first contract (the caller) must decode return values.
+
+Whenever a call is done on a contract type, this will intiate an external call.
+Note that ``this`` returns the current contract, so ``this.foo()`` will do an
+external call, which is much more expensive than ``foo()``.
+
+Passing accounts on Solana on external calls
```

Comment:
```suggestion
Passing accounts on Solana during external calls
```
I just wanted to avoid the repetition of `on`. Feel free to change the preposition, if `during` does not sound good.

Reply:
changed it to with